src/Microsoft.TemplateEngine.Edge/Template/CreationResultStatus.cs

```diff
@@ -1,16 +1,48 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
+
+using System;
+
 namespace Microsoft.TemplateEngine.Edge.Template
 {
+    /// <summary>
```

Comment:
very similar to `New3CommandStatus`

Reply:
`CreationResultStatus` defines exit statuses only for **template creation** operation. Having same statuses as exit codes for new3 command was limiting us for a while, as new3 command supports wider variety of operations, not only template creation, and other operations (install for example) have different kind of errors. 
I split them to two enums here to remove not needed dependencies and we will work on fixing statuses returned by CLI using `New3CommandStatus` in different PR: since enum is now separated and internal, we can add as many statuses as needed instead of trying to find best possible match. I will create separate issue for that.